browser/src/control/jsdialog/Widget.Combobox.js

```diff
@@ -183,6 +183,12 @@ JSDialog.combobox = function (parentContainer, data, builder) {
 	var container = L.DomUtil.create('div', 'ui-combobox ' + builder.options.cssClass, parentContainer);
 	container.id = data.id;
 
+	var labelText = L.Map.StateChangeHandler.prototype.Labels[data.command];
```

Comment:
in general - JSDialog widgets should be built only based on data available inside JSON.
For correct behavior on updates, etc. We need them to be stateless.

Reply:
Occurs that the stateless does not have label info data, so If you read the task description,
I am doing a short path (hack) to resolve the problem.